app/models/box.rb

```diff
@@ -0,0 +1,8 @@
+class Box < ApplicationRecord
+  belongs_to :box_request
+  belongs_to :designed_by
+  belongs_to :design_reviewed_by
+  belongs_to :assembled_by
+  belongs_to :shipped_by
+  belongs_to :shipping_payment
```

Comment:
I pulled this locally and could not `#create!` a box with a nil in any of :designed_by :design_reviewed_by, :assembled_by, :shipped_by, or :shipping_payment. These should all nullable

Reply:
Right!